packages/opentelemetry-cloud-monitoring-exporter/package.json

```diff
@@ -39,11 +39,11 @@
   "devDependencies": {
     "@types/mocha": "7.0.2",
     "@types/nock": "11.1.0",
-    "@types/node": "12.12.47",
     "@types/sinon": "9.0.4",
     "codecov": "3.7.0",
-    "gts": "2.0.2",
+    "gts": "1.1.0",
     "mocha": "7.2.0",
+    "nock": "12.0.3",
```

Comment:
Was this just missing altogether before? Also the one below has `^`, should they be consistent?

Reply:
Before nock was imported from @types/nock. I could be misremembering but I believe this gave me an error when using node 8.

Reply:
The reason for using this specific version is that it was working in opentelemetry-js/opentelemetry-resources which heavily uses nock successfully.